src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,84 @@
+.calendar {
+  box-sizing: content-box;
```

Comment:
move to * selector

Reply:
When removed "box-sizing" in  calendar block it crush 
![image](https://user-images.githubusercontent.com/111124988/234031253-6d48023c-4a0e-4f44-8d0b-ed7d8140e261.png)

Reply:
Mentor meant that you need to write this style to * selector 
like
```
* {
  box-sizing: border-box;
}
```

Reply:
Yes, when I  write this style to * selector, it no work